cmd/cork/downloadimage.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/coreos/mantle/Godeps/_workspace/src/github.com/spf13/cobra"
+	"github.com/coreos/mantle/sdk"
+)
+
+var (
+	downloadImageCmd = &cobra.Command{
+		Use:   "download-image",
+		Short: "Download and verify CoreOS images",
+		Long:  "Download and verify current CoreOS images to a local cache.",
+		Run:   runDownloadImage,
+	}
+	downloadImageRoot         string
+	downloadImageCacheDir     string
+	downloadImagePrefix       string
+	downloadImageVerify       bool
+	downloadImagePlatformList platformList
+)
+
+func init() {
+	downloadImageCmd.Flags().StringVar(&downloadImageRoot,
+		"root", "http://storage.core-os.net/coreos/amd64-usr/master/", "base URL of images")
+	downloadImageCmd.Flags().StringVar(&downloadImageCacheDir,
+		"cache-dir", "", "local dir for image cache")
```

Comment:
For the sake of consistency how about make this default to a directory under `sdk.RepoCache()`